libs/search-widget/src/widgets/suggestions/Suggestions.svelte

```diff
@@ -60,5 +60,6 @@
     text-align: right;
     font-size: small;
     margin: 0;
+    color: #707070;
```

Comment:
use a `var` (define new ones in `wdiget.svelte` if this color is not existing yet)